aries_cloudagent/multitenant/admin/routes.py

```diff
@@ -1,5 +1,6 @@
 """Multitenant admin routes."""
 
+from aries_cloudagent.multitenant.base import BaseMultitenantManager
```

Comment:
This, and other instances should be a relative import and sorted consistently with the other imports. I'll clean it up after merging (since I can't amend this PR).